sensorlib/ReadMe.md

```diff
@@ -0,0 +1,88 @@
+# CARLA Sensor Library
+
+## Overview
+
+The sensor library provides a proxy sensor for the CARLA simulator, exposing additional features.
+
+This new feature is provided to enable vulnerable road user detection using a LIDAR sensor in the context of a
+CARLA-driven simulation. The data is intended to feed into a CARMA V2X system for data fusion with other sensors and
+action by CARMA platform vehicles.
```

Comment:
This data will be feed to CARMA-Platform via the CARLA CARMA Integration tool and to CDASim for any other integrated simulators or software systems under test that need it.